src/store/message-store.ts

```diff
@@ -22,7 +22,7 @@ export interface MessageStore {
   put(
     tenant: string,
     messageJson: BaseMessage,
-    indexes: { [key: string]: string },
+    indexes: Record<string, string>,
```

Comment:
neat!  there's probably a bunch of other places where we could/should do this.  @duncanmak can you file an issue for updating all those too?  :)

Reply:
https://github.com/TBD54566975/dwn-sdk-js/issues/339